tests/data/conservativehq.com-1_simple_article_from_full_page.json

```diff
@@ -1,7 +1,7 @@
 {
   "title": "Beware Soros's \"Rented Evangelicals\"",
   "byline": null,
-  "date": "2018-10-17T00:00:00",
+  "date": null,
```

Comment:
Note, I added the date to this test previously, but we no longer include the site specific xpath for conservativehq and the crawler instead will specify this with the config file